addons/common/functions/fnc_ragdoll_animChangedEH.sqf

```diff
@@ -0,0 +1,119 @@
+/*
+  Author: diwako
+
+  Description:
+  Code for AnimChanged eventhandler
+
+  Parameter(s):
+  _unit - Ragdolled unit
+  _anim - Currently changed animation
+
+  Examples:
+  (begin)
+    [_unit,_anim] call fpa_common_fnc_ragdoll_animChangedEH;
+  (end)
+
+  Returns:
+    none
+*/
+#include "script_component.hpp"
+params ["_unit","_anim"];
+if(!(_unit getVariable ["ACE_isUnconscious",false])) exitWith {}; // do not run if unit is conscious
+if(!(alive _unit) &&  // do not run if unit is dead
+	{!(isNull objectParent _unit)}) exitWith {}; // do not run if unit in any vehicle
```

Comment:
space after if

Reply:
not alive and not in vehicle should be an or right?

Reply:
actually good catch with the wrong AND statement